core/src/main/java/nu/yona/server/subscriptions/entities/Buddy.java

```diff
@@ -113,12 +120,20 @@ public Status getReceivingStatus()
 
 	public void setReceivingStatus(Status status)
 	{
-		getBuddyAnonymized().setReceivingStatus(status);
+		BuddyAnonymized buddyAnonymized = getBuddyAnonymized();
+		buddyAnonymized.setReceivingStatus(status);
+		BuddyAnonymized.getRepository().save(buddyAnonymized);
```

Comment:
The `BuddyAnonymized` was edited here, but not saved anywhere. Could it be the case that this was done automatically by JPA?